sorc/build_workflow_utils.sh

```diff
@@ -5,35 +5,19 @@ set -eux
 readonly UTILS_DIR=$(cd "$(dirname "$($cmd -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
 
 # Adapt for global-workflow structure.
-target=${target:-"NULL"}
+source ${UTILS_DIR}/machine-setup.sh > /dev/null 2>&1
```

Comment:
why is this required?
which mahcine is requiring this to be sourced that supports the global workflow?

Reply:
When I tried running build_workflow_utils stand-alone, $target isn't defined until it is set by machine-setup, so it was using NULL, then quietly not loading a modulefile because there wasn't one for NULL, then cmake would fail.